recipes/sphinxdft/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# MKL patch - requires autoconf
+mkdir -p src/playground
+touch src/playground/Makefile.am
+autoreconf -vif
+rm -rf src/playground
+
+# Setup
+export CXX=$GXX
```

Comment:
Is this line needed?
```suggestion
```
(I think it's the same on Linux and is unset on macOS)

Reply:
The compilation is a bit tricky, the sphinx `configure` script is not able to recognise: 

```
CXX=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++
```

as g++ but it works fine with:

```
GXX=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-g++ 
```

Therefore I had to overwrite the variable.

Reply:
In that case can you leave include a brief comment in the script.